src/app/modules/project/add/manual/manual.component.ts

```diff
@@ -124,6 +126,12 @@ export class ManualComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    /* 
+    * First check user for authentication: if false, they should login. If true, they can continue.
+    */
+    if(!this.authService.isAuthenticated()) {
+      this.router.navigate(['/account/login']);
```

Comment:
The way you did it isn't wrong but it's recommended to use this syntax.
```suggestion
      this.router.navigate(['account', 'login']);
```